tsdb/batcher.go

```diff
@@ -75,33 +88,30 @@ func (b *PointBatcher) Start() {
 			case <-b.stop:
 				if len(batch) > 0 {
```

Comment:
While we're here... this check isn't needed, as it's in `emit()`, and calling `emit()` will force the timer to stop.